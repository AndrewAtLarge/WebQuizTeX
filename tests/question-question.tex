% should give error for nested questions
\documentclass{webquiz}
\begin{document}

  \begin{question}
    \begin{question}

    \end{question}
  \end{question}

\end{document}
