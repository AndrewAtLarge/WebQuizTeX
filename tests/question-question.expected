! Package webquiz Error: question environments cannot be nested!.
! Package webquiz Error: Each question must contain a choice environment or an 
! Package webquiz Error: Each question must contain a choice environment or an 
